chart/load-balancer-api/templates/deployment.yaml

```diff
@@ -91,6 +91,7 @@ spec:
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           args:
             - serve
+            - "--config=/config/loadbalancerapi.yaml"
```

Comment:
feels like `config` by itself could be re-worded to something more specific, like maybe `relationships-config`?

Reply:
I think I'd like to keep it as a generic config and not specific to the relations right now in case people need more complex structures for config like maps/structs